geodepy/tests/test_statistics.py

```diff
@@ -0,0 +1,150 @@
+import unittest
+from geodepy import statistics
+from geodepy.statistics import np
+
+
+class TestStatistics(unittest.TestCase):
+    def test_rotation_matrix(self):
+        lat = 19.4792
+        lon = 70.6931
+
+        expected_result = np.array([
+            [-0.94376114, -0.11025276, 0.31170376],
+            [0.33062805, -0.31471096, 0.88974272],
+            [0.0,  0.94276261,  0.33346463],
+        ])
+
+        result = statistics.rotation_matrix(lat, lon)
+
+        np.array_equal(expected_result, result)
+        self.assertEqual(type(expected_result), type(result))
+
+    def test_vcv_cart2local_and_vcv_local2cart2_3X3(self):
```

Comment:
I think this test can be done better. No realistic VCV array would be all zero, and if it was there would be no need to transform it from Cartesian to local, or vice versa. I think a good test would be to create a realistic Cartesian VCV, transform it to local and then back to Cartesian. The result should match the initial VCV